perceiver/model/core/modules.py

```diff
@@ -376,6 +380,7 @@ def __init__(
         first_self_attention_block_shared: bool = True,
         self_attention_widening_factor: int = 1,
         dropout: float = 0.0,
+        residual_dropout: float = 0.0,
```

Comment:
Please add this parameter to the API docs (only if these already exist) and extend the description of `dropout` to contain "post-attention dropout".